tests/utils/ProtocolV3TestBase.sol

```diff
@@ -467,10 +469,10 @@ contract ProtocolV3TestBase is DiffUtils {
 
     if (localConfig.virtualAccActive) {
       localConfig.virtualBalance = pool.getVirtualUnderlyingBalance(reserve.tokenAddress);
-      localConfig.aTokenUnderlyingBalance = IERC20Detailed(reserve.tokenAddress).balanceOf(
-        localConfig.aToken
-      );
     }
+    localConfig.aTokenUnderlyingBalance = IERC20Detailed(reserve.tokenAddress).balanceOf(
+      localConfig.aToken
```

Comment:
this was still address zero at this place which is why balance was arbitrary (whatever was donated to address zero)